src/main/java/uk/gov/justice/hmpps/prison/api/resource/MovementResource.java

```diff
@@ -13,16 +13,7 @@
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestParam;
-import uk.gov.justice.hmpps.prison.api.model.ErrorResponse;
-import uk.gov.justice.hmpps.prison.api.model.Movement;
-import uk.gov.justice.hmpps.prison.api.model.MovementCount;
-import uk.gov.justice.hmpps.prison.api.model.OffenderIn;
-import uk.gov.justice.hmpps.prison.api.model.OffenderInReception;
-import uk.gov.justice.hmpps.prison.api.model.OffenderMovement;
-import uk.gov.justice.hmpps.prison.api.model.OffenderOut;
-import uk.gov.justice.hmpps.prison.api.model.OffenderOutTodayDto;
-import uk.gov.justice.hmpps.prison.api.model.RollCount;
-import uk.gov.justice.hmpps.prison.api.model.TransferSummary;
+import uk.gov.justice.hmpps.prison.api.model.*;
```

Comment:
Minor point but should not be using wildcard imports (unless it is a unit test)....Intellij setting changes will resolve this (assuming you use this).

Reply:
sorted